source/server/ai/zombie_core.qc

```diff
@@ -1824,13 +1827,13 @@ void(entity where) spawn_a_zombieB =
 	updateLimb (szombie, 2, szombie.rarm);
 
 #endif // FTE
-	
-	
-	szombie.solid = SOLID_CORPSE;
+
```

Comment:
As we discussed off platform recently you can't do this. Vril has unique physics designed to allow for player<->corpse interactions without corpses colliding with each other. Either wrap this change in an ifdef or see if there's a different solution, thanks!